package.json

```diff
@@ -90,7 +90,8 @@
   },
   "pnpm": {
     "overrides": {
-      "cross-fetch": "4.0.0"
+      "cross-fetch": "4.0.0",
+      "node-fetch": "^2.6.13"
```

Comment:
I specified it here so that it overrides `node-fetch` wherever it's used, because I've introduced the same version into `providers/package.json` so that its fetch function can be used as the default one.